### ISSUE TITLE
AttributeError: 'DiGraph' object has no attribute 'node'

### ISSUE THREAD

**Issue by mbaelde** [globality-corp/sklearn-hierarchical-classification#29]
Hi everyone,

I ran into the error while trying to execute the example code `classify_digits.py`:

```
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-7-8c87a92ee2cd> in <module>
----> 1 classify_digits()

<ipython-input-3-787263de3ec1> in classify_digits()
     42     )
     43 
---> 44     clf.fit(X_train, y_train)
     45     y_pred = clf.predict(X_test)
     46 

~/.virtualenv/python3.6/site-packages/sklearn_hierarchical_classification/classifier.py in fit(self, X, y, sample_weight)
    184         # Recursively build training feature sets for each node in graph
    185         with self._progress(total=self.n_classes_ + 1, desc="Building features") as progress:
--> 186             self._recursive_build_features(X, y, node_id=self.root, progress=progress)
    187 
    188         # Recursively train base classifiers

~/.virtualenv/python3.6/site-packages/sklearn_hierarchical_classification/classifier.py in _recursive_build_features(self, X, y, node_id, progress)
    260 
    261         """
--> 262         if "X" in self.graph_.node[node_id]:
    263             # Already visited this node in feature building phase
    264             return self.graph_.node[node_id]["X"]

AttributeError: 'DiGraph' object has no attribute 'node'
```

I installed `sklearn-hierarchical-classification` using `pip install sklearn-hierarchical-classification` on Ubuntu 18.04.3 LTS with Python 3.6.9.

I think this is related to the issue #28. Does anyone has a hot fix for this please ?

**Comment by mbaelde** [comment]
It seems that the proper attribute for `DiGraph` is `nodes` and not `node` in `networkx>=2.0`.

**Comment by adamhadani** [comment]
fixed by #30
